README.md

```diff
@@ -272,3 +272,38 @@ If your fields are not localized, the value of the field will be returned.
 In case of the field being localized but no value being set for a given entry, it will use
 a fallback locale, by default is `en-US` but can be specified as an additional
 parameter in all the mentioned calls.
+
+### Preview API Helper
+
+You can use the `#with_preview` helper to try your Preview API content without having to
+generate the entire `data` structures.
+
+This generates a new Preview Contentful Client and has a cache that will store your objects
+in memory until they are considered to need refresh.
+
+It can be used like a Contentful Client:
+
+```erb
+<% with_preview(space: 'cfexampleapi', access_token: 'b4c0n73n7fu1') do |preview| %>
+  <% entry = preview.entry('nyancat') %>
+
+  <p>Name: <%= entry.name %></p>
+<% end %>
+```
+
+If you want to clear the cache to force a refresh:
+
+```erb
+<% with_preview(space: 'cfexampleapi', access_token: 'b4c0n73n7fu1') do |preview| %>
+  <% preview.clear_cache %>
+<% end %>
+```
+
+#### Caching Rules
+
+* Every preview client will be cached by Space/Access Token combination
+* Only `entry`, `entries`, `asset` and `assets` will be cached
+* Every call will be cached by it's query parameters and ID (if ID is applicable)
+* Each call will be considered stale after 3 tries or 2 hours
```

Comment:
I can reduce the cache time, and also make it configurable, but it didn't make sense to me to re-fetch every few minutes, because you won't be changing all entries every minute

What time period do you consider a good default?

Also, keep in mind that you can manually clear the cache using `preview.clear_cache`